open-metadata-implementation/user-interfaces/presentation-server/nodejs/package.json

```diff
@@ -15,7 +15,7 @@
     "dev": "webpack-dev-server --open"
   },
   "author": "David Radley",
-  "license": "ISC",
+  "license": "Apache-2",
```

Comment:
I think it should be 'Apache-2.0' - but just checkign ...

Reply:
Yes - so needs updating
ref: https://spdx.org/licenses/

Reply:
sorry I should have checked - I assumed the sting in the git issue text was correct. Fixed.